pennylane_lightning_kokkos/src/simulator/StateVectorKokkos.hpp

```diff
@@ -452,36 +450,6 @@ template <class Precision> class StateVectorKokkos {
                   }},
             }
     {
-        expval_funcs_["Identity"] = [&](auto &&wires, auto &&params) {
-            return getExpectationValueIdentity(
-                std::forward<decltype(wires)>(wires),
-                std::forward<decltype(params)>(params));
-        };
-
-        expval_funcs_["PauliX"] = [&](auto &&wires, auto &&params) {
-            return getExpectationValuePauliX(
-                std::forward<decltype(wires)>(wires),
-                std::forward<decltype(params)>(params));
-        };
-
-        expval_funcs_["PauliY"] = [&](auto &&wires, auto &&params) {
-            return getExpectationValuePauliY(
-                std::forward<decltype(wires)>(wires),
-                std::forward<decltype(params)>(params));
-        };
-
-        expval_funcs_["PauliZ"] = [&](auto &&wires, auto &&params) {
-            return getExpectationValuePauliZ(
-                std::forward<decltype(wires)>(wires),
-                std::forward<decltype(params)>(params));
-        };
-
-        expval_funcs_["Hadamard"] = [&](auto &&wires, auto &&params) {
-            return getExpectationValueHadamard(
-                std::forward<decltype(wires)>(wires),
-                std::forward<decltype(params)>(params));
-        };
-
```

Comment:
How is performance affected by this and other associated moves. Can you run some benchmarks on CPU & GPU across 6-27 qubits to see?

Reply:
Not a major impact as expected. Plz check the plots [here](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/47#issue-1627767897).

Reply:
Thanks @maliasadi